client/src/util/api.js

```diff
@@ -1,7 +1,7 @@
 import axios from 'axios';
 
 const api = axios.create({
-    baseURL: 'http://localhost:5000/',
+    baseURL: process.env.REACT_APP_PROD ? process.env.REACT_APP_API_URL : 'http://localhost:5000/', 
```

Comment:
Is this API URL env variable set on Heroku?

Reply:
Yup, this is what the config vars on the heroku dashboard looks like. I tested it real quick with some simple display stuff and it works

![Screen Shot 2021-04-07 at 12 59 38 PM](https://user-images.githubusercontent.com/47611808/113912844-39c9fe00-97a1-11eb-9ad3-9b2b94cc5bea.png)